src/scss/_admonitions.scss

```diff
@@ -125,12 +125,12 @@
   }
 
   &.tip {
-    border-color: rgba(var(--color-admonition-info), 1);
+    border-color: rgba(var(--color-admonition-warning), 1);
```

Comment:
Using the categorization of RTD, this should be "success" I think, instead. 

And so the "hint" just above, should not use "warning", but also "success"